docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,256 @@
+# Elastic Subnets Parameters
+
+Avalanche Permissioned Subnets turn into Elastic Subnet via the `TransformSubnetTx` transaction.
+`TransformSubnetTx` specifies a set of structural parameters for the Elastic Subnet.
+This reference describes these structural parameters and illustrates the constraints they must satisfy.
+
```

Comment:
Q: should the contents https://docs.avax.network/specs/banff-changes#transformsubnettx be copied here for easy reference?

Reply:
Done